lib/schemaUtils.js

```diff
@@ -2571,6 +2575,46 @@ module.exports = {
     return constructedValue;
   },
 
+  /**
+   * Parses media type from given content-type header or media type
+   * from content object into type and subtype
+   *
+   * @param {String} str - string to be parsed
+   * @returns {Object} - Parsed media type into type and subtype
+   */
+  parseMediaType: function (str) {
+    let simpleMediaTypeRegExp = /^\s*([^\s\/;]+)\/([^;\s]+)\s*(?:;(.*))?$/,
```

Comment:
@VShingala why can't we split by slash/semicolon?

Reply:
@abhijitkane Yes it could be done. But regex above also make sure to account whitespaces which can not be present in mediatype but can be present in optional params. Found inspiration for this from [Negotiator](https://github.com/jshttp/negotiator/blob/master/lib/mediaType.js).